tests/test_init.py

```diff
@@ -2,27 +2,184 @@
+        journal = auto_container.connection.run_expect(
+            [0], "journalctl --boot"
+        )
+        assert (
+            "Reached target Multi-User System" in journal.stdout
+        ), "Multi-User target was not reached"
+
+    def test_hostnamectl(self, auto_container, host, container_runtime):
+        """
+        Ensure :command:`hostnamectl` works correctly by asserting expected values
+        """
+
+        hostnamectl = auto_container.connection.run_expect([0], "hostnamectl")
+        # Process the printed values to a string map
+        values = TestSystemd._split_values(hostnamectl.stdout)
+        assert values["Chassis"] == "container", "Chassis mismatch"
+        assert (
+            "SUSE Linux Enterprise Server" in values["Operating System"]
+        ), "Missing SUSE tag in Operating system"
+        assert (
+            values["Virtualization"] == container_runtime.runner_binary
+        ), "Virtualization tag mismatch"
+
+    def test_timedatectl(self, auto_container):
+        """
+        Ensure :command:`timedatectl` works as expected and the container timezone is UTC
+        """
+        output = auto_container.connection.run_expect(
+            [0], "timedatectl"
+        ).stdout
+        assert "Time zone: UTC" in output, "Time zone not set to UTC"
+
+        # Check that the reported timestamp for UTC and local time match the system time
+        def check_timestamp(pattern, timestamp, delta):
+            """Checks the timedatectl output for the given pattern against the given timestamp
+            e.g. use the "Universal time" as pattern and datetime.utcnow() to check for the UTC time
+            """
+            grep = [line for line in output.split("\n") if pattern in line]
+            assert (
+                len(grep) == 1
+            ), f"{pattern} not present in timedatectl output"
+            tsp = (
+                grep[0].strip()[len(pattern) + 2 :].strip()
+            )  # Extract actual timestamp
+            tsp = datetime.datetime.strptime(tsp, "%a %Y-%m-%d %H:%M:%S UTC")
+            assert (
+                abs(tsp - timestamp) < delta
+            ), f"timedatectl diff exceeded for {pattern}"
+
+        check_timestamp(
+            "Universal time",
+            datetime.datetime.utcnow(),
+            datetime.timedelta(seconds=59),
+        )
+        # In the container the Local time is expected to be UTC
+        check_timestamp(
+            "Local time",
+            datetime.datetime.utcnow(),
+            datetime.timedelta(seconds=59),
+        )
+
+    def test_no_loginctl_sessions(self, auto_container):
+        """
+        Ensure :command:`loginctl` contains no logins
+        """
+        loginctl = auto_container.connection.run_expect([0], "loginctl")
+        assert (
+            "No sessions" in loginctl.stdout
+        ), "Assert no sessions are present failed"
+
+    @staticmethod
+    def _split_values(output: str, sep: str = ":") -> dict[str, str]:
```

Comment:
Using `dict` for a dictionary type is not supported with older python versions.
```suggestion
    def _split_values(output: str, sep: str = ":") -> Dict[str, str]:
```

Reply:
Done.